packages/hash/external-services/kratos/hooks/after.registration.jsonnet

```diff
@@ -0,0 +1,4 @@
+function(ctx) {
+  // Provide the entire kratos identity to the HTTP request
+  identity: ctx.identity
+}
```

Comment:
nit:
newline

Reply:
Good spot, addressed in https://github.com/hashintel/hash/pull/934/commits/62ddae1cf96aabfc4bb1c7abc05cb78bc29a680a